### PR TITLE
UHF-10239 Image gallery paragraph

### PR DESCRIPTION
**Pull request by annadruid**
# [UHF-10239](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10239)

## What was done

* Created an image gallery paragraph which can be added to basic page upper and lower content areas as well as landing page.

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-10239 `
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-10239`
 * Update the HDBT Admin theme
    * `composer require drupal/hdbt_admin:dev-UHF-10239 `
* Run `make drush-updb drush-cr`

## How to test

* [x] The image gallery should be automatically be added to basic page upper and lower content areas and on landing page to the content area. Make sure it's available in all three. The image gallery will be added to instance related content types in separate pr:s after HDBT and platform config have been merged.
* [x] Add image galleries to a basic page and add different variations to upper and lower content areas.
* [ ] Make sure that in the upper area there are two images next to each other and in the lower area there are three
* [x] Try out the different aspect ratios of the image gallery
* [x] Test that the photographer comes from the image media entity below the image.
* [ ] Test that the design is responsive and works in each breakpoint
* [x] Try out translating the paragraph, it should work normally
* [x] Test that the validation works, meaning there should be at least two gallery items always. It should give an error if there's only one item in a galley.
* [ ] Check that code follows our standards

In etusivu instance:
* [ ] Test that image gallery is available in news content type (see instructions on etusivu pr)

In KYMP
* [ ] Test that image gallery is available in district content type (see instructions on kymp pr)

## Continuous documentation

* [ ] This feature has been documented/the documentation has been updated
* [ ] This change doesn't require updates to the documentation

## Translations

* [ x] Translations have been added to .po -files and included in [this](https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/840) PR

## Other PRs

* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/840
* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/310
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/759
* https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/966

[UHF-10239]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10239?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/scss/06_components/paragraphs/_image-gallery.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  @include breakpoint($breakpoint-l) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.node--type-page,
+.service--full,
+.unit--full {
```
Comment (teroelonen) [inline src/scss/06_components/paragraphs/_image-gallery.scss:17]: This might need to have node--type-district as well.

**Review by teroelonen — CHANGES_REQUESTED** [review]
Might need small adjusting but looks really good so far!

File: src/scss/06_components/paragraphs/_image-gallery.scss (hunk 1 above)
Comment (annadruid) [reply src/scss/06_components/paragraphs/_image-gallery.scss]: Updated these!

**Review by teroelonen — APPROVED** [review]
This is nice 🦖

**Pull request merged**